Mathlib/Data/ZMod/Module.lean

```diff
@@ -10,35 +10,49 @@ import Mathlib.Algebra.Module.LinearMap
 # The `ZMod n`-module structure on Abelian groups whose elements have order dividing `n`
 -/
 
-variable {n : ℕ} {M M₁ : Type*} [AddCommGroup M] [AddCommGroup M₁]
-  [Module (ZMod n) M] [Module (ZMod n) M₁]
```

Comment:
We usually use `M` for modules, even if they're additive groups. Can you change this back to the old names?

(That also has the advantage of making the diff much smaller)

Reply:
Thanks, nice catch, fixed! (I'd accidentally thought "M" stood for "Monoid" - in my own code 😅)